frontend/src/metabase/nav/containers/Navbar.jsx

```diff
@@ -11,23 +11,18 @@ import { t } from "ttag";
 import * as Urls from "metabase/lib/urls";
 import { color, darken } from "metabase/lib/colors";
 
-import EntityMenu from "metabase/components/EntityMenu";
 import Icon from "metabase/components/Icon";
 import Link from "metabase/core/components/Link";
 import LogoIcon from "metabase/components/LogoIcon";
-import Modal from "metabase/components/Modal";
-
-import CollectionCreate from "metabase/collections/containers/CollectionCreate";
-import CreateDashboardModal from "metabase/components/CreateDashboardModal";
 import { AdminNavbar } from "../components/AdminNavbar";
 import PopoverWithTrigger from "metabase/components/PopoverWithTrigger";
+import ProfileLink from "metabase/nav/components/ProfileLink";
+import { ProfileLinkContainer } from "metabase/App.styled";
```

Comment:
Can we move `ProfileLinkContainer` to `Navbar.styled.tsx` since it's not used in the `App` component anymore?